GUACAMOLE-837: Add RDP keymap for Hungarian keyboard layout

P-Zs opened the pull request:
Please add these changes to code base to add Hunagarian keyboard support in Guacamole RDP client

necouchman commented:
Similar to comments on the other pull requests you opened:
- JIRA issue tag should be all caps (GUACAMOLE-837)
- Commit messages should also contain the JIRA issue
- Commit messages should be more descriptive.
- You have two commit messages that contain the same message and presumably update the same file - possibly these should be squashed together?

P-Zs commented:
Thank you for your remarks!

Sorry for my inexperience, I just started usin git and github 3 days ago.
I changed the pull request name to all caps.
I made file edits through github interface. I think the sytem added these commit messages automaticaly, since i don't remember adding these.
The two identical commit message is because i made a mistake in first edit, and modified same filea second time too. If I already made a Pull request can I still merge (squash) these thogeter? Or I need to recreate them somehow? (I already know how to modify the commit messages)

Regards,
Zsolt
